### DISCUSSION TITLE
[META] Plugin Directory Submissions

### DISCUSSION THREAD

**Discussion by homotechsual** [DocusaurusCommunity/website#3] [Show and tell]
## Process Change

**As the plugin directory is now running on the enhanced showcase plugin the process for submitting a new plugin has changed.**

**Please follow [the documentation](https://docusaurus.community/contributing/plugins/) to add a new plugin.**

### Add your plugin to the Docusaurus.community plugin directory

🦕 You built a useful Docusaurus v2+ plugin and want to share it or you found an **open source** plugin which is missing?

🙏Please add it to our community plugin directory — it only takes a few seconds!⏱

### Submit your site

Just post a comment below in this GitHub discussion with some basic information:

Plugin name: <= 60 characters , don't include "Docusaurus Plugin" in the name.
Plugin description: ≤120 characters
Plugin website: link to your plugin documentation / example site (or your repository if you don't have a docs site!)
Plugin source URL: link to your plugin's repository.
Tags: see below the list of available tags.
Author: The author name for the plugin.
Maintained: Boolean indicating whether the plugin is maintained or not.
NPM Packages: Array of package names required for installation.
Minimum Version: The semver string of the minimum compatible Docusaurus version.

You can also just give us the site URL and we'll figure out the rest but providing the other pieces of info remains appreciated and ensure the correct info get displayed 😄.

We will take the plugin website screenshot ourselves.

This data will be used to render a plugin directory card.

### Requirements

We ask your site to respect these mandatory requirements:

It is a production-ready plugin.

To ensure our showcase remains usable over time, we reserve the right to reject lower quality or forked plugins.

### After your submission

We'll get your plugin added as soon as possible.

Please just wait — we'll answer later to avoid creating too many notifications for everyone. You can still unsubscribe if the notification bothers you — we will ping you when replying.

Once a week, we'll add all valid submissions to our site showcase in one batch.

In case your submission is invalid, we will tell you.

### Plugin tags

The list of available plugin tags:

- favorite: Our favourite plugins, don't bother using this - it'll be removed - these are added based on community consensus.
- search: Plugins which add or enhance search functionality.
- api: Plugins which provide features to document APIs.
- utility: Plugins which provide new utility functionality to Docusaurus.
- content: Plugins providing added content types, layouts, or methodologies.
- theme: Plugins providing new theme layers or enhancing additional theming capabilities.
- markdown: Plugins providing new MarkDown syntax enhancements or interpreter capabilities.
- analytics: Plugins providing integration with analytics systems.
- integration: Plugins linking up Docusaurus to external services / systems.

### Notes

- Help us keep the showcase up to date: tell us when a site has the wrong data.
- You can ask for an update of your site showcase data too!
- Technically, you could open a PR yourself and edit [`/src/data/plugins.tsx`](https://github.com/homotechsual/docusaurus.community/edit/main/src/data/plugins.tsx). However, apart from small edits, we prefer to handle the PR ourselves for several reasons (too many PRs, time-consuming reviews, resizing images, fixing site tags and linter issues...)
- We keep the right to edit your submission before publication
- We'll keep this discussion clean over time: submissions will be marked as resolved, and later deleted

**Comment by jy95** [comment]
Plugin name: JSON Schema Viewer / Editor
Plugin description: `A Docusaurus v2 plugin that provides a JSON Schema viewer / editor React component !`
Plugin website: https://jy95.github.io/docusaurus-json-schema-plugin/
Plugin source URL: https://github.com/jy95/docusaurus-json-schema-plugin
Tags: api / theme / content (I suppose)
Author: jy95
Maintained: true
NPM Packages: `docusaurus-json-schema-plugin`
Minimum Version: `^2.2.0`

**Reply by homotechsual** [reply]
Adding now.

**Comment by scottgallant** [comment]
Plugin name: Tinasaurus
Plugin description: Docusaurus starter project with support for TinaCMS (a Markdown editing UI)
Plugin website: https://tina.io/blog/tinasaurus-docusaurus-starter/
Plugin source URL: https://github.com/tinacms/tinasaurus
Tags: markdown, utility, integration, theme
Author: scottgallant
Maintained: true
NPM Packages: `tinacms, @tinacms/cli`
Minimum Version: `2.0.1`

**Comment by schneegansm** [comment]
Plugin name: Markprompt 
Plugin description: A Docusaurus plugin to add LLM-powered docs search, powered by [Markprompt](https://markprompt.com)
Plugin website: [https://markprompt.com/docs#docusaurus](https://markprompt.com/docs#docusaurus)
Plugin source URL: [https://github.com/motifland/markprompt-js/tree/main/packages/docusaurus-theme-search](https://github.com/motifland/markprompt-js/tree/main/packages/docusaurus-theme-search)
Tags: Search, API, Integration, Markdown
Author: Markprompt 
Maintained: true 
NPM Packages: `@markprompt/docusaurus-theme-search`
Minimum Version: `2.2.0`

**Comment by gracefullight** [comment]
- Plugin name: @gracefullight/docusaurus-plugins
- Plugin description: A collection of script plugins to enhance your Docusaurus documentation website.
- Plugin website: https://github.com/gracefullight/docusaurus-plugins
- Plugin source URL: https://github.com/gracefullight/docusaurus-plugins/tree/main/packages
- Tags: Docusaurus, Utility, Integration, Analytics, Chat, Ads
- Author: gracefullight
- Maintained: True
- NPM Packages: @gracefullight/docusaurus-plugin-baidu-analytics, @gracefullight/docusaurus-plugin-channelio, @gracefullight/docusaurus-plugin-chatra, @gracefullight/docusaurus-plugin-cloudflare-analytics, @gracefullight/docusaurus-plugin-gitter, @gracefullight/docusaurus-plugin-google-adsense, @gracefullight/docusaurus-plugin-intercom, @gracefullight/docusaurus-plugin-meta-pixel, @gracefullight/docusaurus-plugin-microsoft-clarity, @gracefullight/docusaurus-plugin-naver-analytics, @gracefullight/docusaurus-plugin-sentry, @gracefullight/docusaurus-plugin-vercel-analytics, @gracefullight/docusaurus-plugin-yandex-metrica
- Minimum Version: ^2

**Comment by alewolf** [comment]
* Plugin name: Cody AI widget
* Plugin description: The plugin adds the Cody AI widget to your Docusaurus website.
* Plugin website: https://github.com/sweetcode-com/docusaurus-plugin-cody/#readme
* Plugin source URL: https://github.com/sweetcode-com/docusaurus-plugin-cody/
* Tags: `utility`
* Author: Aleksandar Vucenovic, aleksandar@sweetcode.com
* Maintained: `true`
* NPM Packages: `[]`
* Minimum Version: `^2.0.0`

**Comment by edno** [comment]
Plugin name: GraphQL-Markdown
Plugin description: Flexible documentation generator for GraphQL schemas.
Plugin website: https://graphql-markdown.github.io/
Plugin source URL: https://github.com/graphql-markdown/graphql-markdown
Tags: `api`, `content`
Author: Gregory Heitz (@edno)
Maintained: `true`
NPM Packages: `@graphql-markdown/docusaurus`, `graphql`, `@graphql-tools/url-loader`, `@graphql-tools/graphql-file-loader`
Minimum Version: `^2.0.0`

**Reply by edno** [reply]
@homotechsual - Do you still maintain this list?

> Once a week, we'll add all valid submissions to our site showcase in one batch.

**Reply by homotechsual** [reply]
Sorry, that was lifted from the original source, the Docusaurus site showcase, this is being refactored and will become self-submission soon. Yes, it's maintained and I'll get this caught up in the next week as I finish the rewrite which will pull each plugin from YAML files (one-per-plugin!)

**Comment by alvarolorentedev** [comment]
Plugin name: Docusaurus Scheduler
Plugin description: A docusaurus compatible GitHub action to schedule blog article releases.
Plugin website: https://github.com/alvarolorentedev/docusaurus-scheduler
Plugin source URL: https://github.com/alvarolorentedev/docusaurus-scheduler
Tags: utility, integration
Author: The author name for the plugin.
Maintained: true
NPM Packages: NA
Minimum Version: ^2.0.0

**Comment by johannesheesterman** [comment]
Plugin name: Codedoc
Plugin description: A plugin for Docusaurus that lets you  write guides on larger chunks of code and highlights the relevant parts.
Plugin website: https://github.com/Elfsquad/docusaurus-plugin-codedoc
Plugin source URL: https://github.com/Elfsquad/docusaurus-plugin-codedoc
Tags: content
Author: Elfsquad
Maintained: true
NPM Packages: @elfsquad/docusaurus-plugin-codedoc
Minimum Version: ^3

**Comment by lebalz** [comment]
Plugin name: Docusaurus Live Brython
Plugin description: Execute and interact with your Markdown Python Codeblocks in Docusaurus 3
Plugin website: https://lebalz.github.io/docusaurus-live-brython/
Plugin source URL: https://github.com/lebalz/docusaurus-live-brython
Tags: content, theme, markdown, integration
Author: Lebalz
Maintained: true
NPM Packages: `docusaurus-live-brython`
Minimum Version: ^3

**Comment by chanjunren** [comment]
Plugin name: Vaultusaurus
Plugin description: Remark / Docusaurus plugin for integrating an Obsidian Vault with a Docusaurus website
Plugin website: https://github.com/chanjunren/vaultusaurus/wiki
Plugin source URL: https://github.com/chanjunren/vaultusaurus
Tags: remark, markdown, integration, theme, utility
Author: Chan Jun Ren
Maintained: True
NPM Packages:   vaultusaurus
Minimum Version: ^3

**Comment by CrazyOrr** [comment]
Plugin name: SSG Fetch
Plugin description: A Docusaurus plugin that allows you to fetch data while doing static site generation
Plugin website: https://github.com/CrazyOrr/docusaurus-plugin-ssg-fetch
Plugin source URL: https://github.com/CrazyOrr/docusaurus-plugin-ssg-fetch
Tags: utility, content
Author: CrazyOrr
Maintained: true
NPM Packages: `docusaurus-plugin-ssg-fetch`
Minimum Version: ^2

**Comment by mirnawong1** [comment]
Plugin name: Autocopy link
Plugin description: One-click header link copying with visual feedback. Click any h2/h3/h4 header to copy its permalink to clipboard.
Plugin website: https://github.com/mirnawong1/autocopy-link
Plugin source URL: https://github.com/mirnawong1/autocopy-link
Tags: utility
Author: mirnawong1
Maintained: true
NPM Packages: ["autocopy-link"]
Minimum Version: 3.0.0

**Reply by mirnawong1** [reply]
hey @homotechsual , just submitted my plugin and wanted to make sure if this was still the right place? many thanks! 🙏

**Comment by tobico** [comment]
The plugin "TypeDoc API" should be updated to indicate that it is no longer maintained. The author has stated in the Readme that the project is no longer actively maintained, and in my experience it doesn't work at all with the latest TypeScript version.

**Comment by guicara** [comment]
Plugin name: Panzoom
Plugin description: Pan & zoom images, SVGs, or HTML elements in Docusaurus. Great for diagrams & complex Mermaid renders.
Plugin website: https://r74tech.github.io/docusaurus-plugin-panzoom/
Plugin source URL: https://github.com/r74tech/docusaurus-plugin-panzoom
Tags: utility, markdown
Author: @r74tech
Maintained: true
NPM Packages: https://www.npmjs.com/package/@r74tech/docusaurus-plugin-panzoom
Minimum Version: 3.6.0

**Comment by guicara** [comment]
Hi @homotechsual,
Are you still maintaining this repository?
Thank youü

**Reply by homotechsual** [reply]
Yes, still maintained - just mid refactor of how the plugin directory works.

**Reply by guicara** [reply]
Thanks for your fast reply.
Great to know you are still maintaining this repo 👍

**Comment by gl0bal01** [comment]
Plugin name: Multi RSS Feed Aggregator
Plugin description: Aggregate and display multiple RSS feeds with category filtering, batch processing, and TypeScript support
Plugin website: https://github.com/gl0bal01/docusaurus-plugin-multi-rss
Plugin source URL: https://github.com/gl0bal01/docusaurus-plugin-multi-rss
Live Demo: [gl0bal01.com/news](https://gl0bal01.com/news)
Tags: content, integration, utility
Author: [gl0bal01](https://github.com/gl0bal01)
Maintained: true
NPM Packages: ["docusaurus-plugin-multi-rss"]
Minimum Version: 3.0.0

**Comment by gl0bal01** [comment]
Plugin name: docusaurus-plugin-obsidian-vault
Plugin description: A Docusaurus plugin that seamlessly integrates your Obsidian vault with Docusaurus, automatically transforming Obsidian-specific syntax (wikilinks, callouts) into Docusaurus-compatible markdown.
Plugin website: https://github.com/gl0bal01/docusaurus-plugin-obsidian-vault
Plugin source URL: https://github.com/gl0bal01/docusaurus-plugin-obsidian-vault
Live Demo: [gl0bal01.com/intel-codex](https://gl0bal01.com/intel-codex)
Tags: content, integration, utility
Author: [gl0bal01](https://github.com/gl0bal01)
Maintained: true
NPM Packages: ["docusaurus-plugin-obsidian-vault"]
Minimum Version: 3.0.0, Typescript

**Comment by FlyNumber** [comment]
Plugin name: Markdown Source
Plugin description: Expose documentation as raw .md URLs for LLMs and AI tools. One-click copy or view markdown with clean output optimized for AI assistants.
Plugin website: [https://github.com/FlyNumber/markdown_docusaurus_plugin](https://github.com/FlyNumber/markdown_docusaurus_plugin)
Plugin source URL: [https://github.com/FlyNumber/markdown_docusaurus_plugin](https://github.com/FlyNumber/markdown_docusaurus_plugin)
Live Demo: [https://www.flynumber.com/docs/](https://www.flynumber.com/docs/)
Tags: content, documentation, ai, developer-tools
Author: FlyNumber
Maintained: true
NPM Packages: ["docusaurus-markdown-source-plugin"]
Minimum Version: 3.0.0

**Comment by tipsxBase** [comment]
Plugin name: ImageViewer
Plugin description: A plugin for Docusaurus v2 that supports image preview using viewerjs！
Plugin website: https://github.com/tipsxBase/docusaurus-plugin-image-viewer
Plugin source URL: https://github.com/tipsxBase/docusaurus-plugin-image-viewer
Tags: image / viewer
Author: tipsxBase
Maintained: true
NPM Packages: docusaurus-plugin-image-viewer
Minimum Version: ^1.0.4

**Comment by CoffeeCupTechWriting** [comment]
Plugin name: Structured Data.
Plugin description: A  plugin that automatically generates JSON-LD structured data (schema.org) for your documentation site's SEO
Plugin website: https://www.npmjs.com/package/@coffeecup_tech/docusaurus-plugin-structured-data
Plugin source URL: https://github.com/CoffeeCupTechWriting/docusaurus-plugin-structured-data
Tags: seo, json-ld, schema.org, metadata, structured-data, i18n
Author: CoffeeCupTechWriting
Maintained: true
NPM Packages: ["@coffeecup_tech/docusaurus-plugin-structured-data"]
Minimum Version: 3.0.0

**Comment by whoisltd** [comment]
Plugin name: Graph View.
Plugin description: An interactive, knowledge graph to your Docusaurus site. It visualizes the relationships between your docs through frontmatter `tags`.
Plugin website: https://www.npmjs.com/package/docusaurus-plugin-graph-view
Plugin source URL: https://github.com/whoisltd/docusaurus-plugin-graph-view
Tags: search, utility
Author: whoisltd
Maintained: true
NPM Packages: ["docusaurus-plugin-graph-view"]
Minimum Version: 3.0.0

**Comment by MrShasha** [comment]
Plugin name: Docucraft
Plugin description: Visual components for Docusaurus documentation. From zoomable diagrams and interactive class graphs to repository maps, we strive to visualize complex information in a way that is human‑comprehensible.
Plugin website: [Documentation website for Docucraft](https://mrshasha.github.io/docucraft/)
Plugin source URL: [Link to GitHub for Docucraft](https://github.com/MrShasha/docucraft)
Tags: content, utility
Author: Jiří Šašek
Maintained: true
NPM Packages: ["docucraft", "react", "react-dom", "@docusaurus/theme-mermaid"]
Minimum Version: 3.0.0

**Comment by asyntai** [comment]
Plugin name: Asyntai AI Chatbot
Plugin description: Adds an AI chat assistant that answers visitor questions from your own documentation, in 80+ languages.
Plugin website: https://www.npmjs.com/package/docusaurus-plugin-asyntai
Plugin source URL: https://github.com/asyntai/docusaurus-plugin-asyntai
Tags: chat, support, ai, analytics
Author: Asyntai
Maintained: true
NPM Packages: ["docusaurus-plugin-asyntai"]
Minimum Version: 2.0.0

**Reply by homotechsual** [reply]
This is no longer how new plugins are submitted. Please check the plugin directory for information.
